config/client/webpack.development.js

```diff
@@ -11,6 +11,8 @@ const webpackCommon = require('./webpack.common');
 module.exports = webpackMerge(webpackCommon, {
   devtool: 'inline-source-map',
 
+  mode: 'development',
```

Comment:
I've removed it on #24 in package json because it is no being used in the app. I think it is not on the webpack config loader on the first place. Please create an issue for that if we will use pug as default. Thanks